packages/rum-core/src/domain/view/viewMetrics/trackLargestContentfulPaint.spec.ts

```diff
@@ -102,4 +100,23 @@ describe('trackLargestContentfulPaint', () => {
 
     expect(lcpCallback).not.toHaveBeenCalled()
   })
+
+  it('should be discarded if it has a size inferior to the previous LCP entry', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about adding a test to ensure that we are notifying multiple times when the size is bigger than the previous entry?